pylint_odoo/test_repo/no_odoo_module/eval_used.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+def func2(param):
```

Comment:
This was extracted from original test of pylint.
Here we can fails all good practices.
But we dont have problem to change it
What is a good name for you in this case?